src/sdk/sdk.ts

```diff
@@ -87,9 +93,17 @@ export class FusionSDK {
             throw new Error('quoter has not returned quoteId')
         }
 
+        let nonce = params.nonce
+
+        // in case of auto request from node
+        if (!params?.nonce || params.nonce === OrderNonce.Auto) {
```

Comment:
Don't need to check for `!params?.nonce`. Just check for `params.nonce === OrderNonce.Auto`